README.md

```diff
@@ -44,4 +75,166 @@ The number at the end is ID of the product you wish to view.
 Product IDs range from 1 - 99.
 
 ### Tests
-Jest is currently configured for testing, but tests do not currently work.
\ No newline at end of file
+Jest is currently configured for testing, but tests do not currently work.
+
+## Usage
+
+With this module you can review reviews by hostel listing. You can also interact with the data using the API.
```

Comment:
Maybe use another word than `review` for `review reviews`.

Reply:
Good catch, thanks.